toolkit/scripts/incremental_building.mk

```diff
@@ -78,6 +78,7 @@ REBUILD_TOOLS                   ?= n
 DELTA_BUILD                     ?= n
 CLEAN_TOOLCHAIN_CONTAINERS      ?= y
 MAX_CPU                         ?=
+PACKAGE_BUILD_TIMEOUT           ?= 4h
```

Comment:
I'd start with something even longer (8 hours?) for the first attempt. Our pipelines build agents are recently faster, so `rust` doesn't take forever, but we don't know, what other people/teams are using.

Reply:
Done